site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -425,34 +425,69 @@ cat <<EOF | kubectl apply -f -
+After the certificates are generated, we will want to [inject caBundle](https://cert-manager.io/docs/concepts/ca-injector/) into controller and extensions webhook and disable controller and extensions secret creation by using yaml file. This process avoids the cluter of passing too many parameters to helm install command. Please refer below `file_name.yaml` file:
 
+```
+agones:
+  controller:
+    disableSecret: true
+    customCertSecretPath:
+    - key: ca.crt
+      path: ca.crt
+    - key: tls.crt
+      path: server.crt
+    - key: tls.key
+      path: server.key
+    allocationApiService:
+      annotations:
+        cert-manager.io/inject-ca-from: agones-system/my-release-agones-cert
+      disableCaBundle: true
+    validatingWebhook:
+      annotations:
+        cert-manager.io/inject-ca-from: agones-system/my-release-agones-cert
+      disableCaBundle: true
+    mutatingWebhook:
+      annotations:
+        cert-manager.io/inject-ca-from: agones-system/my-release-agones-cert
+      disableCaBundle: true
+  extensions:
+    disableSecret: true
+    customCertSecretPath:
+    - key: ca.crt
+      path: ca.crt
+    - key: tls.crt
+      path: server.crt
+    - key: tls.key
+      path: server.key
+    allocationApiService:
+      annotations:
+        cert-manager.io/inject-ca-from: agones-system/my-release-agones-cert
+      disableCaBundle: true
+    validatingWebhook:
+      annotations:
+        cert-manager.io/inject-ca-from: agones-system/my-release-agones-cert
+      disableCaBundle: true
+    mutatingWebhook:
+      annotations:
+        cert-manager.io/inject-ca-from: agones-system/my-release-agones-cert
+      disableCaBundle: true
+```
+
+After, creating yaml file use below command to install Agones:
 ```bash
-helm install my-release \
-  --set agones.controller.disableSecret=true \
-  --set agones.controller.customCertSecretPath[0].key='ca.crt',customCertSecretPath[0].path='ca.crt'
-  --set agones.controller.customCertSecretPath[1].key='tls.crt',customCertSecretPath[1].path='server.crt'
-  --set agones.controller.customCertSecretPath[2].key='tls.key',customCertSecretPath[2].path='server.key'
-  --set agones.controller.allocationApiService.annotations={'cert-manager.io/inject-ca-from': 'agones-system/my-release-cert'} \
-  --set agones.controller.allocationApiService.disableCaBundle=true \
-  --set agones.controller.validatingWebhook.annotations={'cert-manager.io/inject-ca-from': 'agones-system/my-release-cert'} \
-  --set agones.controller.validatingWebhook.disableCaBundle=true \
-  --set agones.controller.mutatingWebhook.annotations={'cert-manager.io/inject-ca-from': 'agones-system/my-release-cert'} \
-  --set agones.controller.mutatingWebhook.disableCaBundle=true \
-  --namespace agones-system --create-namespace  \
-  agones/agones
+helm install my-release --namespace agones-system --create-namespace --values /path/file_name.yaml agones/agones
```

Comment:
```suggestion
helm install my-release --namespace agones-system --create-namespace --values values.yaml agones/agones
```

Reply:
Thanks for suggestions! I have updated the PR.